virt/monitoring/virt-runbooks.adoc

```diff
@@ -7,9 +7,9 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :!virt-runbooks:
-Runbooks for the {VirtProductName} Operator are maintained in the link:https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[openshift/runbooks] Git repository, and you can view them on GitHub. To diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts], follow the procedures in the runbooks.
+To diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts], follow the procedures in the runbooks for the {VirtProductName} Operator. {VirtProductName} alerts are displayed in the *Virtualization* -> *Overview* tab in the web console.
```

Comment:
@jherrman 
All firing alerts are at `observe ->  alerts`
Even-though they can also be found in  `Virtualization* -> *Overview` , I think we should replace this to the general OCP alerts page.

Reply:
Thank you, updated now.